composer.json

```diff
@@ -13,28 +16,28 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "roave/security-advisories": "dev-master",
+        "zendframework/zend-component-installer": "^0.7.0",
```

Comment:
I've removed here `^1.0 ||` because we are still on 0.X versions, so next version can contain BC Breaks.